spec/requests/applications/applications_request_spec.rb

```diff
@@ -37,7 +37,7 @@
 end
 
 feature 'Show application' do
-  let :app do
+  given :app do
```

Comment:
I'v read the Rspec's source code, `given` is just an alias for `let`, and it's match for `feature`